app/build.gradle.kts

```diff
@@ -186,7 +186,6 @@ tasks.register("jacocoTestReport", JacocoReport::class) {
         "**/*Test*.*",
         "android/**/*.*",
         "**/SignatureChecks.*",
-        "**/*Preview*.*",
```

Comment:
Is there a particular reason why this was removed? I thought that it could still help filter out the `"**/*Preview*.*"` files.

Reply:
I took it out because there are no Preview files. I originally added this because I thought it would remove preview _functions_ from coverage, but that was not the case.